### ISSUE TITLE
Unable to open other items because of errors later in the tree

### ISSUE THREAD

**Issue by jipclaassens** [ObjectVision/GeoDMS#706]
![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/3450e0ef-6d35-46bb-8808-026c677a50c0)

**Steps to reproduce: (TL:DR -> scroll to bottom)** 
`C:\ProjDir\Jip\_Github_issues\NetworkModel_PBL_RegressieTest_issue706\cfg\main.dms`

If unlinked data is not present:
`/RegressieTest/Step1_prepare_data`

If present, continue with:
`/RegressieTest/t405_2_NetworkModel_PBL_indicator_results_test/result`

This gives an error
`Link_Node2_rel: out of range or undefined values not allowed`

What is really wrong here? So continue with F2. You'll end up here:
`/NetworkSetup/ConfigurationPerRegio/all/PublicTransport/At_07h00m00s/TransitionTypes/Org2Stop_W/Transition/nr_OrgEntity`

This is not the ultimate item? Manually click on parent 'Transition' and via calculation rule continue to:
`/NetworkSetup/ConfigurationPerRegio/all/PublicTransport/ODConnections/CreateODconnections/Sparse_uitwerking_Org2Stop_W/AB/FromTime_Walk`

The item distance on top is also red. Still with the same error message. Continue F2 until
`/NetworkSetup/ConfigurationPerRegio/all/PublicTransport/ODConnections/RoadNets_Org2Stop_W/Network/ODMatrix/alt_imp`

This is a result of an impedance_matrix. But the 'duration' in the parent container is also red. But the geometry in that container is not accessible because of the later errors:
`/NetworkSetup/ConfigurationPerRegio/all/PublicTransport/ODConnections/RoadNets_Org2Stop_W/Network/geometry`

**TL:DR**
Activate:
`/RegressieTest/t405_2_NetworkModel_PBL_indicator_results_test/result`

Then try to activate 
`/NetworkSetup/ConfigurationPerRegio/all/PublicTransport/ODConnections/RoadNets_Org2Stop_W/Network/geometry`

Why am I not allowed to open this? With Alt+R and activing the mentioned geometry (and the other items in the unit) is working!

![image](https://github.com/ObjectVision/GeoDMS/assets/48675272/cdabbac5-a589-4522-b2ec-d0dbf16003be)

**Comment by MaartenHilferink** [comment]
Fixed ?
![image](https://github.com/ObjectVision/GeoDMS/assets/2284361/a837aa92-6596-4eb8-8348-f45cf1983e04)
and then:
![image](https://github.com/ObjectVision/GeoDMS/assets/2284361/e28a5f72-547a-4563-b6e6-ea42d2ba2393)

**Comment by jipclaassens** [comment]
Not reproducable/maybe fixed.
